examples/cpp_execution/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# ==============================================================================
+# Copyright (c) 2023 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause-Clear 
```

Comment:
license marking should not be clause-clear. Please check other files for reference